### PR TITLE
PCHR-459: Update standard timeline and activity types for built-in case types

### PR DESCRIPTION
**Pull request by omarabuhussein**
I've updated the standard timeline and activity types for build-in case types according to this wiki page :

https://compucorp.atlassian.net/wiki/pages/viewpage.action?pageId=75857955

### CODE REVIEW

File: hrcase/xml/case/Appraisal.xml [hunk 1]
```diff
@@ -60,36 +65,23 @@
...
+          <name>Complete contract revisions</name>
           <reference_offset>0</reference_offset>
         </ActivityType>
-	<ActivityType>
-          <name>Follow up on progress</name>
-	  <reference_offset>30</reference_offset>
-          <reference_select>newest</reference_select>
-        </ActivityType>	
-	<ActivityType>
-          <name>Collection of Appraisal forms</name>
```
Comment (davialexandre) [inline hrcase/xml/case/Appraisal.xml]: Shouldn't these be also removed from the ActivityTypes node above?

File: hrcase/xml/case/Exiting.xml [hunk 2]
```diff
@@ -59,20 +58,22 @@
         </ActivityType>
```
Comment (davialexandre) [inline hrcase/xml/case/Exiting.xml]: The document you linked on the description says the "Schedule Exit Interview" offset should be -10, but you didn't change it (it's 0 now)

Comment (davialexandre) [inline hrcase/xml/case/Exiting.xml]: I just noticed that, different from the wiki page, the ticket description says it should be 0. So I don't know what value it should actually be

**Comment by davialexandre** [comment]
The wiki page linked on the descriptions says Appraisals and Probation don't have any Roles, but there are roles in the XML files. Is that correct?

File: hrcase/xml/case/Appraisal.xml (hunk 1 above)
Comment (omarabuhussein) [inline hrcase/xml/case/Appraisal.xml]: No I've checked with mike and he told me to leave them for now.

File: hrcase/xml/case/Exiting.xml (hunk 2 above)
Comment (omarabuhussein) [inline hrcase/xml/case/Exiting.xml]: indeed , thanks for the catch

**Comment by omarabuhussein** [comment]
> The wiki page linked on the descriptions says Appraisals and Probation don't have any Roles, but there are roles in the XML files. Is that correct?

Yes ... I've check this with mike too and we will keep them for now.

**Comment by davialexandre** [comment]
Approved after the offset fix for the "Schedule Exit Interview" activity type

**Pull request merged**